micro-deps/micro-deps-spring-test-config/src/main/groovy/com/ofg/infrastructure/base/MvcWiremockIntegrationSpec.groovy

```diff
@@ -24,12 +24,12 @@ import static com.ofg.config.BasicProfiles.TEST
 @CompileStatic
 @ContextConfiguration(classes = [MockServerConfiguration])
 @ActiveProfiles(TEST)
-class MvcWiremockIntegrationSpec extends MvcIntegrationSpec {
+abstract class MvcWiremockIntegrationSpec extends MvcIntegrationSpec {
+
+    @Autowired protected HttpMockServer httpMockServer
+    @Value('${wiremock.url:localhost}') protected String wiremockUrl
```

Comment:
This is no longer valid after https://github.com/4finance/micro-infra-spring/pull/196, just hard code `localhost`